src/test/java/seedu/ccacommander/logic/commands/UnenrolCommandTest.java

```diff
@@ -0,0 +1,157 @@
+
+        Enrolment enrolmentToFindOne = TypicalEnrolments.ALICE_AURORA;
+        Enrolment enrolmentToFindTwo = TypicalEnrolments.GEORGE_GRAVITY_DISCOVERY_DAY;
+
+        enrolmentList.add(enrolmentToFindOne);
+        enrolmentList.add(enrolmentToFindTwo);
+
+        assertEquals(enrolmentToFindOne, UnenrolCommand.findEnrolmentFromList(enrolmentList,
+                TypicalMembers.ALICE.getName(), TypicalEvents.AURORA_BOREALIS.getName()));
+        assertEquals(enrolmentToFindTwo, UnenrolCommand.findEnrolmentFromList(enrolmentList,
+                TypicalMembers.GEORGE.getName(), TypicalEvents.GRAVITY_DISCOVERY_DAY.getName()));
+    }
+
+    @Test
+    public void findEnrolmentFromList_fail() {
+        List<Enrolment> enrolmentList = new ArrayList<>();
+
+        Enrolment enrolmentToFindOne = TypicalEnrolments.ALICE_AURORA;
+        Enrolment enrolmentToFindTwo = TypicalEnrolments.GEORGE_GRAVITY_DISCOVERY_DAY;
+
+        enrolmentList.add(enrolmentToFindOne);
+        enrolmentList.add(enrolmentToFindTwo);
+
+        try {
+            UnenrolCommand.findEnrolmentFromList(enrolmentList,
+                    TypicalMembers.CARL.getName(), TypicalEvents.BOXING_DAY.getName());
+        } catch (EnrolmentNotFoundException ee) {
+            assertEquals("Enrolment cannot be found", ee.getMessage());
+        }
+    }
+
+    @Test
+    public void equals() {
+        UnenrolCommand unenrolFirstCommand = new UnenrolCommand(INDEX_FIRST_MEMBER, INDEX_FIRST_EVENT);
+        UnenrolCommand unenrolSecondCommand = new UnenrolCommand(INDEX_SECOND_MEMBER, INDEX_SECOND_EVENT);
+
+        // same object -> returns true
+        assertTrue(unenrolFirstCommand.equals(unenrolFirstCommand));
+
+        // same values -> returns true
+        UnenrolCommand unenrolFirstCommandCopy = new UnenrolCommand(INDEX_FIRST_MEMBER, INDEX_FIRST_EVENT);
+        assertTrue(unenrolFirstCommand.equals(unenrolFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(unenrolFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(unenrolFirstCommand.equals(null));
+
+        // different member -> returns false
+        assertFalse(unenrolFirstCommand.equals(unenrolSecondCommand));
+    }
+
+    @Test
+    public void toStringMethod() {
+        Index memberIndex = Index.fromOneBased(1);
+        Index eventIndex = Index.fromOneBased(1);
+        UnenrolCommand unenrolCommand = new UnenrolCommand(memberIndex, eventIndex);
+        String expected = UnenrolCommand.class.getCanonicalName() + "{memberIndex=" + memberIndex
+                + ", eventIndex=" + eventIndex + "}";
+        assertEquals(expected, unenrolCommand.toString());
+    }
+
+    /**
+     * Updates {@code model}'s filtered enrolment list to show no one.
+     */
+    private void showNoEnrolment(Model model) {
+        model.updateFilteredEnrolmentList(p -> false);
+
+        assertTrue(model.getFilteredEnrolmentList().isEmpty());
+    }
```

Comment:
What is the use of this here?

Reply:
I saw that the `DeleteMemberCommand` and `DeleteEventCommand` also had this test so I reckon that `UnenrolCommand` will require this test too as it involves the deletion of an entity. Should I remove this test case? I'm thinking that this test case is meant to check if the`updateFilteredEnrolmentList` method is working as intended.

Reply:
Ah I think right now it is a function and not a test since it's lacking an @test

Reply:
Alright, will add in the missing annotation!